### ISSUE TITLE
[Angular 6] Jest test fails when constructor param is typed

### ISSUE THREAD

**Issue by dereklin** [thymikee/jest-preset-angular#156]
I have just upgraded to Angular 6, before that, my tests were running fine with jest.

My repo is at https://github.com/dereklin/nrwl-nx-workspace-demo/tree/feature/upgrade-nrwl-6

The test in question is at https://github.com/dereklin/nrwl-nx-workspace-demo/blob/feature/upgrade-nrwl-6/libs/session-resolver/src/lib/session-resolver.spec.ts

When I run 

```
npm run test libs/session-resolver/src/lib/session-resolver.spec.ts
```
It works.

The code being tested is at https://github.com/dereklin/nrwl-nx-workspace-demo/blob/feature/upgrade-nrwl-6/libs/session-resolver/src/lib/session-resolver.ts

Here is the constructor:

```
constructor(@Inject(HTTPSERVICE) private simpleHttpService: any) {}
```
Notice the type is any.

When I change the type to HttpService:
```
constructor(@Inject(HTTPSERVICE) private simpleHttpService: HttpService) {}
```
When I run the test, I get:

```
 FAIL  libs\session-resolver\src\lib\session-resolver.spec.ts
  ● Test suite failed to run

    Error: No message was provided

Test Suites: 1 failed, 1 total
```

When I run the test using karma:
```
ng test --project=session-resolver
```
It works as expected.

Any ideas?

**Comment by dereklin** [comment]
fixed by upgrading to "jest-cli": "23.0.0" which gave me the errors to fix the problems.  It's unrelated to the preset.

**Comment by manojp1988** [comment]
I too had this issue when I upgraded my project from v5 to v7.  During upgrade process tsconfig.spec.json file was updated with silly errors. After I have corrected this error, tests ran fine.
